### ISSUE TITLE
[REQUEST] Reduce trap door frame thickness.

### ISSUE THREAD

**Issue by cloudrambler** [tsunamayo/Starship-EVO#3420]
The trap door frame is shown as 1 brick thick, but has a bounding box 2 bricks thick. There is a specific benefit to reducing the bounding box to the visible thickness; it would allow the trap door to be used as a maintenance "door" into a 3 brick wide passage. Presently the player character can move in a 3 brick wide passage, but the door frame blocks this.

To reproduce:

Place an item of big equipment on a floor surface.
Make an access passage with a brick wall in the adjoining voxel, leaving 3 block widths to walk through.
Note that the player character can move freely in this space (it's ideal for sealed-off maintenance access as it's just 1 voxel wide).
Place a trap door in an upright configuration, also opposite the equipment, aligned with the brick wall and opening outward.
Note that the door frame is not visually wider than the brick wall.
Try to walk through the door and into the passage; the frame seems to prevent movement sideways.

**Comment by tsunamayo** [comment]
Hi, okay sorry mate I am very confusing, but I might be foggy today. I think a picture would help. Sometime you talk about 3 brick wide, sometime 3 block wide, I am confused... Cheers

**Comment by cloudrambler** [comment]
@tsunamayo My bad. I have made an example case to demonstrate the problem.

![20210106190409_1](https://user-images.githubusercontent.com/70821082/103809934-40d25a80-5052-11eb-8cc8-4e9f507ca43e.jpg)

This is an aerial view of an "access passage" next to a shield generator (for example). The passage is three bricks wide and has a wall of one brick layer. Overall the passage fits within a single voxel width, so is nicely compact and efficient. On the floor of the passage I have marked a white area that the player character can walk to from the door (trapdoor), and a red area which is not accessible. This appears to be because the bounding box of the trapdoor is asymmetric and partially blocks the "passage" on one side but not the other.

So my request is to have the trapdoor bounding box adjusted so that movement to both sides is possible.
